app/assets/stylesheets/base/_surveyjs.scss

```diff
@@ -1,21 +1,193 @@
 //--------------------------------------------------
 // variables
 //--------------------------------------------------
-
-//--------------------------------------------------
-// mixins
-//--------------------------------------------------
+$radio-checkbox-width-height: rem-calc(39);
 
 //--------------------------------------------------
 // classes
 //--------------------------------------------------
-.sv-body {
-  font-family: $body-font;
-  min-height: 75vh;
```

Comment:
this was set so the progress bar doesn't jump on navigation is 60vh enough on fullscreen?